week-4-practise/level-1/exercise-1.js

```diff
@@ -38,7 +38,9 @@ var writers = [
     alive: true
   },
 ];
-
+writers.forEach(function (writer) {
+  console.log("Hi, my name is " +writer.firstName+ " " +writer.lastName+". I am "+writer.age+ " years old, and work as a "+writer.occupation);
+});

```

Comment:
Good.  When we later cover template strings, you'll see we can alternatively write this:

```
console.log(`Hi, my name is ${writer.firstName} ${writer.lastName}.  I am ${writer.age} years old, and work as a ${writer.occupation}`);
```